### ISSUE TITLE
Improve docs on passing args to node process

### ISSUE THREAD

**Issue by pellebjerkestrand** [Azure/azure-functions-nodejs-worker#648]
`func start` with `-- "--prof"` seems to do nothing. I, at least, can't find the resulting *.log file anywhere on my system.

Adding `--prof` to `NODE_OPTIONS` in `local.settings.json` results in an error message.

Relevant output snippet:

```sh
Azure Functions Core Tools
Core Tools Version:       4.0.4865 Commit hash: N/A  (64-bit)
Function Runtime Version: 4.12.2.19454

[2022-12-01T13:39:27.380Z] Language Worker Process exited. Pid=14562.
[2022-12-01T13:39:27.380Z] node exited with code 9 (0x9). .
[2022-12-01T13:39:27.390Z] node: --prof is not allowed in NODE_OPTIONS
```

Easy profiling of Node.js apps (https://nodejs.org/en/docs/guides/simple-profiling/) seems impossible, unless I'm missing something.

Please allow us to profile Node.js apps run with `func start`.

**Comment by pellebjerkestrand** [comment]
Same behavior with the most recent version I can get from npm.

```sh
Azure Functions Core Tools
Core Tools Version:       4.0.4895 Commit hash: N/A  (64-bit)
Function Runtime Version: 4.13.0.19486
```

**Comment by ejizba** [comment]
Hi @pellebjerkestrand the recommended way to modify the arguments to the node process is to use the `languageWorkers__node__arguments` setting. Here's at least one example in the docs: https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2-v3-v4-export%2Cv2-v3-v4-done%2Cv2%2Cv2-log-custom-telemetry%2Cv2-accessing-request-and-response%2Cwindows-setting-the-node-version#in-local-development-environment

**Comment by pellebjerkestrand** [comment]
> Hi @pellebjerkestrand the recommended way to modify the arguments to the node process is to use the `languageWorkers__node__arguments` setting. Here's at least one example in the docs: https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2-v3-v4-export%2Cv2-v3-v4-done%2Cv2%2Cv2-log-custom-telemetry%2Cv2-accessing-request-and-response%2Cwindows-setting-the-node-version#in-local-development-environment

Thanks a lot! That was not easy to find.

**Comment by ejizba** [comment]
Yeah I agree it's a bit of a needle in a haystack and we've heard this feedback before. Moving the issue to the Node.js repo for follow up.

Do you have any suggestions of where you may have looked to find this information?

**Comment by pellebjerkestrand** [comment]
It felt natural to me that passing arguments to the node process could be done on the command line, so I was looking in `func start`'s documentation. I found nothing, so I searched for lots of different versions of "azure-functions-core-tools" + "pass argument to node" + "func start" and so on. I found plenty of blog posts and answers on Stack Overflow mentioning _both_ `-- "--prof"` and using `NODE_OPTIONS` in `local.settings.json` with people writing that at least one of those worked for them.

If using `--prof` in `NODE_OPTIONS` gave a hint at what can be done instead (use `languageWorkers__node__arguments`), that would be great.

I still think it's more ergonomic to be able to pass these arguments on the command line when using `func start` since I rarely want profiling to stick around for long (and I guarantee I will forget that it's on and keep it longer than needed).

**Comment by gunzip** [comment]
**@ejizba** According to the [documentation](https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=javascript%2Cwindows%2Cazure-cli&pivots=nodejs-model-v4#languageworkers__node__arguments), using `languageWorkers__node__arguments` can negatively impact cold start times because it forces the runtime to start a new worker from scratch instead of using pre-warmed ones.  

Does setting arguments via `NODE_OPTIONS` have the same drawback?

I'm asking because using OTEL with ESM requires passing specific arguments to Node.js, as mentioned here: https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md#additional-notes-on-experimental-loaders.

Thank you in advance!
